### DISCUSSION TITLE
Podcasts fehlen in der Bibliothek

### DISCUSSION THREAD

**Discussion by Cheater-Bill** [audiamus/BookLibConnect#17] [Ideas]
Hallo Audiamus,
mir ist aufgefallen, dass man im Downloadbereich seine Audible Podcasts garnicht mit sieht, obwohl die zumindest auf der Website auch mit in der Bibliothek angezeigt werden. Wäre cool, wenn du das noch mit einbauen könntest. :) Mein alter Walkman hat leider kein Internet und ist auf MP3 files angewiesen.

Viele Grüße

**Comment by audiamus** [comment]
Podcasts werden vorläufig noch ausgeblendet. Da funktioniert das Herunterladen anders, und das muss ich erst ergründen. Steht auf der Wunschliste, aber nicht mit hoher Priorität.

**Reply by icom962** [reply]
Wäre es möglich, noch verschlüsselte Podcast aax-files (zB von einem Mobilgerät mit AudibleApp) mit BookLibConnect zu entschlüsseln und dann in mp3 zu wandeln?

**Reply by audiamus** [reply]
Wie gesagt, ich habe mich mit Podcasts noch nicht beschäftigt. Dateiendung .aax könnte allerdings auf die klassische Verschlüsselung deuten, für die der statische Schlüssel aus der Audible Win10 App oder dem früheren Audible Manager der passende ist. Dann wäre AAX Audio Converter ausreichend.

**Reply by icom962** [reply]
Ich hatte mich leider getäuscht - nicht .aax, sondern .aaxc; die liegen im download-Bereich der Audible-App auf Android.
Wenn ich die in .aax umbenenne, werden sie von audio converter auch erkannt und die Tags auch richtig extrahiert - der eigentlich (Ton-)INhalt allerdings nicht :(

**Reply by audiamus** [reply]
Dann wird es nicht gehen. AAXC erfordert den individuellen Schlüssel per Datei und Gerät. 

_AAX und AAXC sind immer MP4-Container, und die Metadaten entsprechen dem MP4-Standard, sogar die proprietären. Deswegen kann man die Tags sehen._

**Comment by Cheater-Bill** [comment]
ok vielen Dank :)

**Comment by ClapTrap1305** [comment]
Ich möchte mit diesem Beitrag lediglich auch nur das Interesse meiner Seite für Podcasts bekunden.
Vielleicht pushed es ja etwas das Thema auf der Prioliste bei @audiamus

**Reply by HarveyOs** [reply]
Dem kann ich mich nur anschließen. Ich bin auch sehr an Podcasts interessiert.

Und vielen Dank an @audiamus, ein super Programm, weiter so.

**Comment by kapi21** [comment]
Otra persona aqui interesada. Gracias
